provider/app.py

```diff
@@ -66,6 +66,14 @@ def postTrigger(namespace, trigger):
             'error': "trigger and namespace from route must correspond to triggerURL"
         })
         response.status_code = 409
+    elif body["isMessageHub"] == False and (enable_generic_kafka == False):
```

Comment:
To be more pythonic, we should replace Line 69 with 
elif not body["isMessageHub"] and not enable_generic_kafka: